README.md

```diff
@@ -1,2 +1,66 @@
-# reggaemedia
-the reggaemedia website
+# Getting started
+1. Before starting development server you need to define environment variables.
+2. To be able to deploy project stack, your PATH should include executable aws package with configured IAM account.
+3. Preferred package manager is `yarn` version 2.
+
+## .ENV file
+Environment variables file must include vars written below to be able to deploy AWS stack. Vars below is stale and should be define once for whole project.
+
+```
+COGNITO_CALLBACK_URL=http://localhost:3000/api/auth/callback/cognito
+ADMIN_EMAIL
+CDK_DEFAULT_ACCOUNT= default account that is being defined in aws configuration
+CDK_DEFAULT_REGION= preferred region
+AMAZON_ACCESS_KEY= IAM credentials
+AMAZON_SECRET_KEY= IAM credentials
+AMAZON_REGION= preferred region
+```
+
+After defining variables above you should run command `yarn stack:deploy`. That command will generate `cdk-outputs.json` that is being gitignored, output file contain another variables that must be defined in the same .env file as above. 
```

Comment:
I suppose this part can be safely removed if we'll integrate this into deployment process